medusa/restore_node.py

```diff
@@ -131,7 +131,10 @@ def restore_node_locally(config, temp_dir, backup_name, in_place, keep_auth, see
         if in_place:
             cassandra.start_with_implicit_token()
         else:
-            cassandra.start(tokens)
+            if medusa.utils.evaluate_boolean(config.kubernetes.enabled):
```

Comment:
This block is already bypassed in the case of k8s due [to this condition](https://github.com/thelastpickle/cassandra-medusa/pull/308/files#diff-558c6e6db1fe2499e845c25c2379d77a10d0529846a0ee9d3e54931cd9b25d2dR122).

I think we rather need to add an else clause: 

```
    if not medusa.utils.evaluate_boolean(config.kubernetes.enabled):
        if seeds is not None:
        ...
        ...
    elif not in_place:
      # kubernetes remote restore
      cassandra.replaceTokensInCassandraYamlAndDisableBootstrap(tokens)
```

What do you think?

Reply:
Ouh, my blind eyes.